hacs.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "Sun Card",
```

Comment:
Agree to stay with the original name. I did add a note about the fork to the repo description!

Would you by any chance know how to add an icon to the project?

![image](https://user-images.githubusercontent.com/2870104/225671830-75699cb0-98e3-4181-81cb-2d4ffc9f32bb.png)

Reply:
I'm pretty sure icons are not available for lovelace.
If you filter hacs list for lovelace items only there's no single one with an Icon.